### ISSUE TITLE
Packaging game throws error

### ISSUE THREAD

**Issue by drewc5131** [20tab/UnrealEnginePython#297]
```
Project.Build: ********** BUILD COMMAND STARTED **********
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\Desktop\DiscordBot\Recode\Py34\Lib\site-packages\PyQt4\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\Desktop\DiscordBot\Recode\Py34\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\Desktop\DiscordBot\Recode\Py34\Scripts\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\Desktop\DiscordBot\Recode\DPython\Lib\site-packages\PyQt4\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\Common Files\Intel\Shared Libraries\redist\intel64\compiler\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\Razer Chroma SDK\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\Razer Chroma SDK\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\system32\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\System32\Wbem\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\System32\WindowsPowerShell\v1.0\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\system32\config\systemprofile\AppData\Local\Microsoft\WindowsApps\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\QuickTime\QTSystem\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\Git\cmd\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\CMake\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Panda3D-1.0.5\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\Microsoft DNX\Dnvm\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\Microsoft SQL Server\130\Tools\Binn\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\dotnet\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\PuTTY\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\Skype\Phone\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\dub\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\D\dmd2\windows\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\nodejs\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\system32\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\System32\Wbem\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\System32\WindowsPowerShell\v1.0\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\Gource\cmd\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\gradle-2.9\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\apache-ant-1.8.2\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\jdk1.8.0_77\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\android-ndk-r12b\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\android-sdk-windows\extras\android\support\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\android-sdk-windows\build-tools\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\android-sdk-windows\platform-tools\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\NVPACK\android-sdk-windows\tools\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\.dnx\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\Downloads\ffmpeg-20160628-c0cb53c-win64-static\ffmpeg-20160628-c0cb53c-win64-static\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\ProgramData\Oracle\Java\javapath\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\Razer Chroma SDK\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files\Razer Chroma SDK\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Windows\system32\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Windows\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Windows\System32\Wbem\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Windows\System32\WindowsPowerShell\v1.0\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\system32\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\System32\Wbem\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\WINDOWS\System32\WindowsPowerShell\v1.0\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\GtkSharp\2.12\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Program Files (x86)\GtkSharp\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Users\Drew\AppData\Local\Microsoft\WindowsApps\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\depot_tools\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\MinGW\bin\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Panda3D-1.10.0-x64\python\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Panda3D-1.10.0-x64\python\Scripts\xgConsole.exe=False
InternalUtils.SafeFileExists: SafeFileExists C:\Panda3D-1.10.0-x64\bi\xgConsole.exe=False
UE4Build.Build: XGE was requested, but is unavailable, so we won't use it.
UE4Build.Build: ************************* UE4Build:
UE4Build.Build: ************************* ForceMonolithic: False
UE4Build.Build: ************************* ForceNonUnity:False
UE4Build.Build: ************************* ForceDebugInfo: False
UE4Build.Build: ************************* UseXGE: False
UE4Build.Build: ************************* UseParallelExecutor: False
InternalUtils.SafeFileExists: SafeFileExists P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe=True
InternalUtils.SafeDeleteFile: SafeDeleteFile P:\Epic Games\Projects\LODB\Intermediate\Build\Manifest.xml
InternalUtils.SafeFileExists: SafeFileExists P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe=True
InternalUtils.SafeFileExists: SafeFileExists P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe=True
CommandUtils.Run: Running: P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe LODB Win64 Development -Project="P:\Epic Games\Projects\LODB\LODB.uproject"  "P:\Epic Games\Projects\LODB\LODB.uproject" -NoUBTMakefiles  -remoteini="P:\Epic Games\Projects\LODB" -skipdeploy -noxge -generatemanifest -NoHotReload
ProcessResult.StdErr:   ERROR: Could not find definition for module 'UnrealEnginePython' (referenced via default plugins -> UnrealEnginePython.uplugin)
CommandUtils.Run: Took 1.5359675s to run UnrealBuildTool.exe, ExitCode=5
ExceptionUtils.PrintExceptionInfo: ==============================================================================
ExceptionUtils.PrintExceptionInfo: ERROR: Command failed (Result:5): P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe LODB Win64 Development -Project="P:\Epic Games\Projects\LODB\LODB.uproject"  "P:\Epic Games\Projects\LODB\LODB.uproject" -NoUBTMakefiles  -remoteini="P:\Epic Games\Projects\LODB" -skipdeploy -noxge -generatemanifest -NoHotReload. See logfile for details: 'UnrealBuildTool-2018.01.27-12.57.59.txt' 
ExceptionUtils.PrintExceptionInfo:        (see C:\Users\Drew\AppData\Roaming\Unreal Engine\AutomationTool\Logs\P+Epic+Games+UE_4.18\UAT_Log.txt for full exception trace)
ExceptionUtils.PrintExceptionInfo: 
ExceptionUtils.PrintExceptionInfo: CommandFailedException: Command failed (Result:5): P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe LODB Win64 Development -Project="P:\Epic Games\Projects\LODB\LODB.uproject"  "P:\Epic Games\Projects\LODB\LODB.uproject" -NoUBTMakefiles  -remoteini="P:\Epic Games\Projects\LODB" -skipdeploy -noxge -generatemanifest -NoHotReload. See logfile for details: 'UnrealBuildTool-2018.01.27-12.57.59.txt' 
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.CommandUtils.RunAndLog(String App, String CommandLine, String Logfile, Int32 MaxSuccessCode, String Input, ERunOptions Options, Dictionary`2 EnvVars, SpewFilterCallbackType SpewFilterCallback)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.CommandUtils.RunAndLog(CommandEnvironment Env, String App, String CommandLine, String LogName, Int32 MaxSuccessCode, String Input, ERunOptions Options, Dictionary`2 EnvVars, SpewFilterCallbackType SpewFilterCallback)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.CommandUtils.RunUBT(CommandEnvironment Env, String UBTExecutable, String CommandLine, String LogName, Dictionary`2 EnvVars)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.CommandUtils.RunUBT(CommandEnvironment Env, String UBTExecutable, FileReference Project, String Target, String Platform, String Config, String AdditionalArgs, String LogName, Dictionary`2 EnvVars)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.UE4Build.BuildWithUBT(String TargetName, UnrealTargetPlatform TargetPlatform, String Config, FileReference UprojectPath, Boolean ForceMonolithic, Boolean ForceNonUnity, Boolean ForceDebugInfo, Boolean ForceFlushMac, Boolean DisableXGE, String InAddArgs, Boolean ForceUnity, Dictionary`2 EnvVars)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.UE4Build.Build(BuildAgenda Agenda, Nullable`1 InDeleteBuildProducts, Boolean InUpdateVersionFiles, Boolean InForceNoXGE, Boolean InUseParallelExecutor, Boolean InForceNonUnity, Boolean InForceUnity, Boolean InShowProgress, Dictionary`2 PlatformEnvVars, Nullable`1 InChangelistNumberOverride, Dictionary`2 InTargetToManifest)
ExceptionUtils.PrintExceptionInfo:    at Project.Build(BuildCommand Command, ProjectParams Params, Int32 WorkingCL, ProjectBuildTargets TargetMask)
ExceptionUtils.PrintExceptionInfo:    at BuildCookRun.DoBuildCookRun(ProjectParams Params)
ExceptionUtils.PrintExceptionInfo:    at BuildCookRun.ExecuteBuild()
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.BuildCommand.Execute()
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.Automation.Execute(List`1 CommandsToExecute, Dictionary`2 Commands)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.Automation.Process(String[] Arguments)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.Program.MainProc(Object Param)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.InternalUtils.RunSingleInstance(Func`2 Main, Object Param)
ExceptionUtils.PrintExceptionInfo:    at AutomationTool.Program.Main()
ExceptionUtils.PrintExceptionInfo: ==============================================================================
Program.Main: AutomationTool exiting with ExitCode=5 (5)
```

**Comment by unbit** [comment]
Are you using a source distribution ? You cannot package with a binary one

**Comment by drewc5131** [comment]
no, that would explain, i didnt fully read that part of the readme. my bad

**Comment by drewc5131** [comment]
After doing that, doing all the proper steps, the editor crashes on loading
![image](https://user-images.githubusercontent.com/31282527/35475875-07721efc-0374-11e8-8967-0765d79d361e.png)

**Comment by rdeioris** [comment]
Hi, it generally means it cannot find the python dll. Very probably you were using a binary embedded distribution with python included in the plugin. You need to install python in your system (adding it to the PATH) or downloading a binary distribution and copy it into the Binaries directory

**Comment by drewc5131** [comment]
got it

**Comment by drewc5131** [comment]
Still doesnt work...
```UATHelper: Packaging (Windows (64-bit)): ********** BUILD COMMAND STARTED **********
UATHelper: Packaging (Windows (64-bit)): Running: P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe LODB Win64 Development -Project="P:\Epic Games\Projects\LODB\LODB.uproject"  "P:\Epic Games\Projects\LODB\LODB.uproject" -NoUBTMakefiles  -remoteini="P:\Epic Games\Projects\LODB" -skipdeploy -noxge -generatemanifest -NoHotReload
UATHelper: Packaging (Windows (64-bit)):   Using Python at: C:/Panda3D-1.10.0-Py3.5-x64/python
UATHelper: Packaging (Windows (64-bit)): Took 1.5119521s to run UnrealBuildTool.exe, ExitCode=0
UATHelper: Packaging (Windows (64-bit)): Running: P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe LODB Win64 Development -Project="P:\Epic Games\Projects\LODB\LODB.uproject"  "P:\Epic Games\Projects\LODB\LODB.uproject" -NoUBTMakefiles  -remoteini="P:\Epic Games\Projects\LODB" -skipdeploy -noxge -NoHotReload -ignorejunk
UATHelper: Packaging (Windows (64-bit)):   Using Python at: C:/Panda3D-1.10.0-Py3.5-x64/python
UATHelper: Packaging (Windows (64-bit)):   Performing 3 actions (4 in parallel)
UATHelper: Packaging (Windows (64-bit)):   Module.UnrealEnginePython.2_of_3.cpp
UATHelper: Packaging (Windows (64-bit)):   P:\Epic Games\Projects\LODB\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Private\UEPyEngine.cpp(164): warning C4996: 'FPaths::GameContentDir': FPaths::GameContentDir() has been superseded by FPaths::ProjectContentDir(). Please update your code to the new API before upgrading to the next release, otherwise yo
ur project will no longer compile.
UATHelper: Packaging (Windows (64-bit)):   p:\epic games\ue_4.18\engine\source\runtime\core\public\Misc/Paths.h(137): note: see declaration of 'FPaths::GameContentDir'
UATHelper: Packaging (Windows (64-bit)):   P:\Epic Games\Projects\LODB\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Private\UEPyUScriptStruct.cpp(98): error C2039: 'HasMetaData': is not a member of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   p:\epic games\ue_4.18\engine\source\runtime\coreuobject\public\UObject/UnrealType.h(71): note: see declaration of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   P:\Epic Games\Projects\LODB\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Private\UEPyUScriptStruct.cpp(100): error C2039: 'GetMetaData': is not a member of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   p:\epic games\ue_4.18\engine\source\runtime\coreuobject\public\UObject/UnrealType.h(71): note: see declaration of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   P:\Epic Games\Projects\LODB\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Private\UEPyUScriptStruct.cpp(141): error C2039: 'HasMetaData': is not a member of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   p:\epic games\ue_4.18\engine\source\runtime\coreuobject\public\UObject/UnrealType.h(71): note: see declaration of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   P:\Epic Games\Projects\LODB\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Private\UEPyUScriptStruct.cpp(143): error C2039: 'GetMetaData': is not a member of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   p:\epic games\ue_4.18\engine\source\runtime\coreuobject\public\UObject/UnrealType.h(71): note: see declaration of 'UProperty'
UATHelper: Packaging (Windows (64-bit)):   ERROR: UBT ERROR: Failed to produce item: P:\Epic Games\Projects\LODB\Plugins\UnrealEnginePython\Binaries\Win64\UE4-UnrealEnginePython.lib
UATHelper: Packaging (Windows (64-bit)):   Total build time: 39.38 seconds (Local executor: 0.00 seconds)
UATHelper: Packaging (Windows (64-bit)): Took 39.7721998s to run UnrealBuildTool.exe, ExitCode=5
UATHelper: Packaging (Windows (64-bit)): ERROR: Command failed (Result:5): P:\Epic Games\UE_4.18\Engine\Binaries\DotNET\UnrealBuildTool.exe LODB Win64 Development -Project="P:\Epic Games\Projects\LODB\LODB.uproject"  "P:\Epic Games\Projects\LODB\LODB.uproject" -NoUBTMakefiles  -remoteini="P:\Epic Games\Projects\LODB" -skipdeploy -noxge -NoHotReload -ignoreju
nk. See logfile for details: 'UnrealBuildTool-2018.01.27-15.33.56.txt' 
UATHelper: Packaging (Windows (64-bit)):        (see C:\Users\Drew\AppData\Roaming\Unreal Engine\AutomationTool\Logs\P+Epic+Games+UE_4.18\UAT_Log.txt for full exception trace)
UATHelper: Packaging (Windows (64-bit)): AutomationTool exiting with ExitCode=5 (5)
UATHelper: Packaging (Windows (64-bit)): BUILD FAILED```

**Comment by rdeioris** [comment]
Hi, use the version from github. The src zip in the release section does not build on non-editor 4.18
